src/OutdatedLibraries/ComposerOutdated.php

```diff
@@ -45,15 +46,10 @@ public function execute(InputInterface $input, OutputInterface $output): int
                 }
             }
         }
-
-        array_unshift($tab, '| Chronos (API) | version  | version disponible |', '| --- | --- | --- |');
+        $header = $this->generateHeader('API');
```

Comment:
```suggestion
        $header = $this->generateHeader('Chronos (API)');
```
En plus de ça tu peux directement faire `$tab = $this->generateHeader('Chronos (API)');` ligne 26